CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -550,6 +551,8 @@ public enum Keyword: CaseIterable {
       return KeywordSpec("higherThan")
     case .if:
       return KeywordSpec("if", isLexerClassified: true)
+    case .implementation:
+      return KeywordSpec("implementation")
```

Comment:
Could you mark the keyword as experimental, eg. like `_borrowing`. For that you’ll need to plumb the experimental feature through.
- Define it in https://github.com/apple/swift-syntax/blob/main/CodeGeneration/Sources/SyntaxSupport/ExperimentalFeatures.swift
- Map it in https://github.com/apple/swift/blob/f81f7f9c544abb362f148589bc18609591c0952d/lib/ASTGen/Sources/ASTGen/SourceFile.swift#L53

Reply:
Sure, will do.